client/src/www/app/outline_server_repository/server.ts

```diff
@@ -45,6 +49,15 @@ export class OutlineServer implements Server {
         this.sessionConfig = staticKeyToShadowsocksSessionConfig(accessKey);
         break;
     }
+    if (!_name) {
+      if (this.sessionConfigLocation) {
```

Comment:
How would we handle agnostic config in the future? I am thinking of letting Go return the name to TypeScript?

Reply:
This code is transport-agnostic already. This is the location of the dynamic config. It will stay as a URL, we don't need to worry about it.